__mocks__/redux-persist/integration/react.js

```diff
@@ -0,0 +1,3 @@
+// jest.mock("redux-persist/integration/react", () => ({
```

Comment:
why is the file commented? also its bad practice to push commented code. 
What does the file do?

Reply:
I kept the file so I may debug the setup of the test environment later on. I'll remove it since its bad practice.